packages/esm-form-entry-app/src/app/openmrs-api/program-resource.service.ts

```diff
@@ -59,22 +59,23 @@ export class ProgramResourceService {
       location: locationUuuid,
     };
 
+
     this.isPatientEnrolled(patientUuid, programUuid).subscribe((result: boolean) => {
       if (!result) {
         this.httpClient.post(this.programEnrollmentUrl(), payload).subscribe(
           (response) => {
-            showToast({
+            showSnackbar({
               title: 'Program enrollment',
-              description: 'Patient has been enrolled successfully',
+              subtitle: 'Patient has been enrolled successfully',
```

Comment:
@jona42-ui how was this success message before the change, was it appearing with high contrast(black background)? Because we might need to change it to a low contrast(green background) since the designs expect success messages to be green. cc @denniskigen how should this be handled?